### PR TITLE
Updating required version for IBMQuantumExperience

### PR DESCRIPTION
**Pull request by ajavadia**
Fixes issue #228, taking into account recent API changes for hpc.

## Description

Now requires IBMQuantumExperience-1.8.25 or higher.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ x] All new and existing tests passed.

### CODE REVIEW

File: test/python/test_initializer.py [hunk 1]
```diff
@@ -27,7 +27,6 @@
 from qiskit.tools.qi.qi import state_fidelity
 from .common import QiskitTestCase
 
-
```
Comment (diego-plan9) [inline test/python/test_initializer.py]: Can you reintroduce this newline? It is the only thing [[the linter is complaining about](https://travis-ci.org/QISKit/qiskit-sdk-py/jobs/325310217):
```
test/python/test_initializer.py:30:1: E302 expected 2 blank lines, found 1
```

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-IBMQuantumExperience>=1.8.19
+IBMQuantumExperience>=1.8.25
```
Comment (diego-plan9) [inline requirements.txt]: Can you also update the line on `setup.py`, to keep them in sync?

**Comment by ajavadia** [comment]
Thanks for reviewing @diego-plan9! I made those fixes.

**Comment by diego-plan9** [comment]
Wonderful, thanks @ajavadia ! :+1:

**Pull request merged**